plugins/org.jkiss.dbeaver.ext.exasol/OSGI-INF/l10n/bundle_ja.properties

```diff
@@ -216,8 +216,8 @@ tree.functionsauths.node.name           = \u00A0\u95A2\u6570\u7279\u6A29
 tree.global_metadata.node.name          = \u00A0\u30B0\u30ED\u30FC\u30D0\u30EB\u30E1\u30BF\u30C7\u30FC\u30BF
 tree.locks.node.description             = \u00A0\u30B5\u30FC\u30D0\u30FC\u30ED\u30C3\u30AF\u30DE\u30CD\u30FC\u30B8\u30E3\u30FC
 tree.locks.node.name                    = \u00A0\u30ED\u30C3\u30AF\u30DE\u30CD\u30FC\u30B8\u30E3\u30FC
-tree.procedure.node.name                = \u00A0\u624B\u9806
-tree.procedures.node.name               = \u00A0\u624B\u7D9A\u304D
```

Comment:
Current `procedure` translation is mistranslation. 
Also I fixed other `procedure` translation.